Makefile

```diff
@@ -15,11 +15,7 @@ ATLAS_URIS = "$(ATLAS_FREE)" "$(ATLAS_REPLSET)" "$(ATLAS_SHARD)" "$(ATLAS_TLS11)
 TEST_TIMEOUT = 600
 
 .PHONY: default
-default: check-env check-fmt vet build-examples lint errcheck test-cover test-race
-
-.PHONY: check-env
```

Comment:
This should be added back now that the check-env.sh is back.

Reply:
Done.